src/gui/dialogs/liveadjust_z.cpp

```diff
@@ -192,13 +191,13 @@ LiveAdjustZ::LiveAdjustZ()
 
 void LiveAdjustZ::moveNozzle() {
     uint16_t old_top = nozzle_icon.Top();
-    Rect16 moved_rect = nozzleRect;                     // starting position - 0%
-    float percent = adjuster.GetValue() / z_offset_min; // z_offset value in percent
+    Rect16 moved_rect = nozzleRect;                                                        // starting position - 0%
+    float relative = (z_offset_max - adjuster.GetValue()) / (z_offset_max - z_offset_min); // relative z_offset value
```

Comment:
A `static_assert` making sure that `min != max` to ensure no 0 div would be nice, but it's just a minor nitpick.

Reply:
Then this whole screen wouldn't make sense. I think in that case HardFault will be fine.

Reply:
What can be caught at compile time should be caught at compile time, `static_assert` is essentially free and exactly for making sure that stuff 'makes sense'. However in this case it would require constexpr enabling the constants, which might be effort that's not worth it (however small).